_includes/configuration/dockerized-hyperlinking-service.md

```diff
@@ -61,19 +44,46 @@ and download either:
     The structure of the extracted files will be:
     ```sh
     tinymce-hyperlinking-service/
+    ├── ephox-hyperlinking-docker-starter-kit.zip
+    ├── ephox-hyperlinking.war
+    ├── license.txt
+    ├── readme.txt
+    └── version.txt
+    ```
+4. Navigate into the extracted folder.
+
+    ```sh
+    cd tinymce-hyperlinking-service
+    ```
+{% endif %}
+5. Extract the contents of `ephox-hyperlinking-docker-starter-kit.zip`, such as:
+
+    ```sh
+    unzip ephox-hyperlinking-docker-starter-kit.zip -d hyperlinking-service-dockerfile
```

Comment:
This seems overly complicated? Why not just extract them into the current directory, as then you don't need to move the war file.

Reply:
I want to disagree so I don't have to update this page again, but you're not wrong :) updating now.